sessionState/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
The other examples are gradle subprojects - they don't need their own gradle wrapper, etc. scripts. Instead, you add this example to settings.gradle in the root level.

That also adds a requirement that the CI can actually run your example  - there is a standard target runAll which runs all of the examples. We do that to make sure the examples don't rot and stop working over time.

Reply:
Thanks for the information! I'm currently working on adapting the examples structure so it can be built by the runAll functionality of gradle, but the nature of this example is that the execution of the webapp being built by gradle is meant to be done in an installation of tomcat not included with the demo itself. This means that run time errors (such as if dependencies change for the geode module this relies on) wouldn't be shown in a gradle build. 

For this reason I've specified both a version of geode as well as a version of tomcat to use, although I realize that this is not typically how one of these examples would be structured. If this is a problem then this example would probably need to be reformatted somehow to include a baked in version of tomcat so the runAll behavior could actually execute the demo.